tools/topology/topology2/include/components/multiband_drc/default.conf

```diff
@@ -10,70 +10,70 @@ Object.Base.data."multiband_drc_config" {
 		0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
 		0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
 		0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
-		0x00,0x00,0x00,0x00,0x7b,0x8e,0x88,0xfe,
-		0x36,0x79,0x57,0x17,0x5f,0x19,0x6f,0x06,
-		0xe8,0x79,0x67,0xd4,0x00,0x00,0x00,0x40,
+		0x00,0x00,0x00,0x00,0x59,0xa5,0xa3,0xfc,
```

Comment:
this seems mostly like a no-op change, so I'm curious why the diff here is so large. Do you have any ideas?

Reply:
I think its updated coefficients. @singalsu pls confirm.

Reply:
Yes, I updated the parameters of multiband-DRC default.conf. I think there may be an issue in pre-emphasis/de-emphasis, there's heavy distortion with -1 dBFS test sweep unless I keep the amount of emphasis a tiny fraction of decibel. I wanted to keep the feature "on" to keep it tested and tracked for MCPS. I will look into that later. It may be in the filters scaling, not necessarily in the code.

Reply:
The changes are in this commit: https://github.com/thesofproject/sof/pull/9599/commits/9c4b972b38bc96427f0fc8de5d652c0bfab0310a

Reply:
@johnylin76 please verify

Reply:
I roughly checked the blob content. As 9c4b972 changed the default parameters for DRC and pre-emphasis, the changed bytes look reasonable.